lib/mutant/subject/method.rb

```diff
@@ -28,7 +28,11 @@ def name
       # @api private
       #
       def expression
-        Expression.parse("#{context.identification}#{self.class::SYMBOL}#{name}")
+        Expression::Method.new(
+          scope_symbol: self.class::SYMBOL,
+          scope_name:   scope.name,
+          method_name:  name.to_s
```

Comment:
Why prefer a string for a method name? Wouldn't a `Symbol` be a simpler primitive?